packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -39,9 +45,32 @@ type Slot = {
   };
 };
 
-function parseDocumentReference(
+function getRequestHomeCommunityId(request: OutboundDocumentQueryReq): string {
+  return request.gateway.homeCommunityId;
+}
+
+function getResponseHomeCommunityId(
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  extrinsicObject: any
+): string {
+  return stripUrnPrefix(extrinsicObject?._home);
```

Comment:
what happens when `extrinsicObject?._home` is undefined/null/array/object? isn't this just going to error?

Reply:
I added a line to handle if its undefined, but note that it is a required field:
https://profiles.ihe.net/ITI/TF/Volume2/ITI-38.html#3.38.4.1.2.1
"If returnType='LeafClass' the ExtrinsicObject and RegistryPackage elements shall contain the home attribute."
https://github.com/metriport/metriport/blob/2b7110cb2fdc579ce4ea67ed4fadb05553326732/packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti38-envelope.ts#L51